### PR TITLE
FIX - dealing with issue from email bug report

### PR DESCRIPTION
**Pull request by rksh**

### CODE REVIEW

**Comment by bobzilladev** [comment]
We're you able to dupe the problem to make sure this fixed?

**Comment by rksh** [comment]
unable to reproduce in any environment.

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/TokenAccess.java [hunk 1]
```diff
@@ -97,6 +98,7 @@ public static boolean setApiToken(SharedPreferences prefs, String apiToken) {
                 android.os.Build.VERSION_CODES.JELLY_BEAN_MR2) {
             //ALIBI: no crypto available here
             editor.putString(ListFragment.PREF_TOKEN, apiToken);
+            editor.apply();
```
Comment (rksh) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/TokenAccess.java]: Pre-JB compat issue.

**Comment by bobzilladev** [comment]
this ready to go? we can mergez

**Pull request merged**